src/components/Tabs/Tabs.tsx

```diff
@@ -65,29 +65,20 @@ export const Tabs = ({
   onChange,
   ...other
 }: Props) => {
-  /**
-   * Initialize states, constants, and refs
-   */
   const ref = useRef<number | undefined>();
   const headerRef = useRef<HTMLDivElement>(null);
   const [activeIndexState, setActiveIndexState] = useState(activeIndex);
   const [scrollableLeft, setScrollableLeft] = useState<boolean>(false);
   const [scrollableRight, setScrollableRight] = useState<boolean>(false);
 
-  /**
-   * Set the only children components allowed within <Tabs> to be Tab
-   */
-  const tabs = useCallback<() => React.ReactElement<TabProps>[]>(() => {
```

Comment:
Now that we have working types for `allByType`, we can git rid of the type param passed to `useCallback`.